less/pages/team.less

```diff
@@ -1,56 +1,26 @@
 .team-page {
+  background: #fff;
 
-}
-
-.member-profiles {
-  display: flex;
-  flex-direction: column;
-  margin: -10px 0 0 -10px;
-  flex-wrap: wrap;
-  @media screen and (min-width: 768px) {
-    flex-direction: row;
+  .hero-unit-container + .content {
+    overflow: visible;
+    h1 {
+      margin-top: 2rem;
+    }
+  }
+  .tab-switcher.pull-up {
+    margin-top: -4.93rem;
```

Comment:
It seems to be `hidden-xs-down` class being assigned to the `<span>Tab Name</span>`. It's related to the ticket you created on https://github.com/mozilla/mofo-ui/issues/43 . Until it's fixed on `mofo-ui` I guess I can override the CSS so tab labels still appear on small device.